pkg/object/bos.go

```diff
@@ -61,6 +62,9 @@ func (q *bosclient) Create() error {
 	if err != nil && isExists(err) {
 		err = nil
 	}
+	if q.storageClass != "" && err == nil {
+		err = q.c.PutBucketStorageclass(q.bucket, q.storageClass)
```

Comment:
Can we update the storage class of existing bucket?

Reply:
I think we should not update the storage type of buckets that were not created by us, so I will change the logic of similar code.

Reply:
yes